osbrain/agent.py

```diff
@@ -803,8 +866,7 @@ def kill(self):
 
     def close_sockets(self):
         for address in self.socket:
-            if address in ('loopback', '_loopback_safe',
-                           'inproc://loopback', 'inproc://_loopback_safe'):
+            if self._is_address_internal(address):
```

Comment:
Not sure if it makes sense to create an `_is_address_internal` method just for this `address in (...)` (assuming it is not used anywhere else).